taf/TAF_Closure.py

```diff
@@ -953,16 +960,13 @@ def sum_paid(
         cond4="=1",
         paidcol="tot_mdcd_pd_amt",
         outcol="",
-        ):
+    ):
         """
         Sum tot_mdcd_pd_amt on the headers OR mdcd_pd_amt on the lines where the given column equals the given condition.
         """
-         
 
         if not outcol:
-            _outcol = condcol1
-        else:
```

Comment:
Why was this part removed? It seems like this changes functionality that wouldn't be related to this ticket.

Reply:
Fair question! For some reason the code was dead. The _outcol is not being used anywhere in the code. Beyond this it should say:
```if not outcol:
            outcol = condcol1```

Now. I believe that should be checked-in as it's in my branch.

Reply:
I see. I looked around and all calls to `sum_paid` all have a non-empty string `outcol` so this should be ok. I'd think `_outcol` is a private variable, but it just seems to be used elsewhere as a way to have 2 local variables with similar names.